script/create-redirects.js

```diff
@@ -8,7 +8,7 @@ function getRedirectPage(redirectToPath) {
   return `
     <!doctype html>
     <head>
-      <meta http-equiv=refresh content="1; url=${absolutePath}">
+      <meta http-equiv=refresh content="0; url=${absolutePath}">
```

Comment:
I had that `1` there only to reveal momentarily that we had a link that should be updated, although that isn't an option for sites outside of ours.

Reply:
I can revert that back. From what I read, shorter intervals there made it more likely that you wouldn't lose some of the link ranking in search engines. But given that this is a fallback, maybe that's not necessary.